### ISSUE TITLE
Difference in betwen Error,Warning and Notices.

### ISSUE THREAD

**Issue by reach2akram** [squizlabs/HTML_CodeSniffer#113]
I am using HTML Code_Sniffer for WCAG 2.0 AA. Could you please let me know what should I cover that is Error+Warning+Notices or Error+Warnings or Only Errors ?

**Comment by gsherwood** [comment]
Start by trying to fixed all the errors. These are things that violate the WCAG standard and HTML_CodeSniffer is confident about checking. If you can remove all these, you'll greatly improve the accessibility of your site.

If you want to go a bit further, try and fix the warnings. These will help with the accessibility of your site, but aren't as important as the errors. Still, I like to try and remove all these as well.

Notices are things that provide information to you so you can make your own decision. These are things HTML_CodeSniffer can't check automatically, so it is good to review them but most people ignore them. You'll also never be able to remove the notices because HTML_CodeSniffer can't determine if you have "fixed" something.

So my advice is to try and remove all errors + warnings, but focus more on the errors than anything else.
